packages/components/src/FilterDateRangePicker/subcomponents/FilterDateRangePickerField/hooks/useEndDateValidation.spec.tsx

```diff
@@ -51,7 +51,7 @@ describe("useEndDateValidation()", () => {
           startDate: new Date("2023-05-01"),
           startDateFieldLabel: "Start date",
         })
-        expect(newDate).toBeUndefined()
+        expect(newDate).toEqual(new Date("2023-04-03"))
```

Comment:
You need to update the test description (also if you could put `()` in the describe that would be great)